### PR TITLE
handle directory symlinks

### PR DESCRIPTION
**Pull request by chrisblossom**
Redo of #18. I was able to overcome the issues I was having previously.

~~I am unsure how to update yarn.lock at this point. Please update this PR with the correct ``yarn.lock`` update for ``fs-extra``.~~

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -83,22 +85,39 @@ function normalizePaths (value) {
...
+    val => val.split(tempDirReal).join('<TEMP_DIR>'),
     val => val.split(tempDir).join('<TEMP_DIR>'),
+
     // Replace home directory with <HOME_DIR>
+    val => val.split(homeDirReal).join('<HOME_DIR>'),
     val => val.split(homeDir).join('<HOME_DIR>'),
+
     // handle HOME_DIR nested inside TEMP_DIR
     val => val.split(`<TEMP_DIR>${path.sep + homeRelativeToTemp}`).join('<HOME_DIR>'),
+    val => val.split(`<TEMP_DIR>${path.sep + homeRelativeToTempReal}`).join('<HOME_DIR>'), // untested
```
Comment (chrisblossom) [inline lib/index.js:148]: If you have an idea how to test these two "untested" lines, please add to this PR or give me an idea how I should. I personally think it is okay to merge without these tested as they are safeguards.

**Comment by chrisblossom** [comment]
I've successfully added ``fs-extra`` and merged changes from master.

File: lib/index.js (hunk 1 above)
Comment (tribou) [reply lib/index.js:148]: Yep, I just ran this patch against a couple projects and all existing functionality is still running fine. Also, you've increased our test suite 10x 🙌 so I don't have a problem merging. If anyone actually encounters this edge-case, we can open a new issue.

**Comment by tribou** [comment]
Published in `v0.1.15`
